spec/lib/tmuxinator/project_spec.rb

```diff
@@ -691,9 +693,11 @@
       before { project.yaml["pane_title_format"] = nil }
 
       it "configures a default format" do
+        # rubocop:disable Lint/InterpolationCheck
         expect(project.tmux_set_pane_title_format).to eq(
           'tmux set pane-border-format "#{pane_index}: #{pane_title}"'
```

Comment:
I think you can appease rubocop by replacing the line with
```suggestion
          "tmux set pane-border-format \"\#{pane_index}: \#{pane_title}\""
```
kinda ugly, but also makes it clear the non-interpolation is intentional.